docs/wiki/protocol/prehistory.md

```diff
@@ -0,0 +1,310 @@
+
+![A statue dedicated to Satoshi, and Bitcoin announcement post.](img/overview/satoshi-and-bitcoin.jpg)
+**A statue dedicated to Satoshi, and Bitcoin announcement post.**
+
+Bitcoin is also the largest socio-economic experiment the world has ever seen:
+
+> "When Satoshi Nakamoto first set the Bitcoin blockchain into motion in January 2009, he was
+> simultaneously introducing two radical and untested concepts. The first is the 'bitcoin', a decentralized
+> peer-to-peer online currency that maintains a value without any backing, intrinsic value or central issuer. So
+> far, the 'bitcoin' as a currency unit has taken up the bulk of the public attention.
+>
+> However, there is also another, equally important, part to Satoshi's grand experiment: the concept of a proof of
+> work-based blockchain to allow for public agreement on the order of transactions."\
+> — Vitalik Buterin
+
+[Several](https://web.archive.org/web/20230404234458/https://www.etoro.com/wp-content/uploads/2022/03/Colored-Coins-white-paper-Digital-Assets.pdf) [attempts](https://en.wikipedia.org/wiki/Namecoin) were made to build applications on top of Bitcoin's network to leverage the newly created digital currency. However, for this purpose Bitcoin's network proved primitive, and the applications were built using complex and not very scalable workarounds.
+
+Ethereum emerged as a solution to address these challenges.
+
+## The Ethereum world computer
+
+In 2012, [Vitalik Buterin](https://en.wikipedia.org/wiki/Vitalik_Buterin) and Mihai Alisie founded [Bitcoin Magazine](https://en.wikipedia.org/wiki/Bitcoin_Magazine) - the first serious publication dedicated to digital currencies. Vitalik soon discovered the limitations of Bitcoin and [proposed a platform](https://web.archive.org/web/20150627031414/http://vbuterin.com/ultimatescripting.html) that would support generalized financial applications.
+
+In 2014, with the help of [Gavin Wood](https://en.wikipedia.org/wiki/Gavin_Wood), the [design of Ethereum was formalized](https://ethereum.github.io/yellowpaper/paper.pdf).
+
+![Vitalik, Jeff, and Gavin working on Ethereum.](img/overview/ethereum-launch.jpg)
+**Vitalik, Jeff, and Gavin working on Ethereum.**
+
+On July 30, 2015, Ethereum [went live](https://etherscan.io/block/1) as a platform aimed at building tools for a self-sovereign economy using digital currency.
+
+As of the time of writing, Ethereum has a market capitalization of **$400 billion.**
+
+> 📄 READ: [Vitalik's post about the origin of Ethereum.](https://vitalik.eth.limo/general/2017/09/14/prehistory.html)
+
+> 🎦 WATCH: [Mario Havel talks about the Ethereum philosophy.](https://streameth.org/ethereum_protocol_fellowship/watch?session=65d77e4f437a5c85775fef9d)
+
+> 📄 READ: [Evolution of Ethereum.](/wiki/protocol/history.md)
+
+## Resources
```

Comment:
I would add recommendation for movie Revolution OS and maybe this https://www.youtube.com/watch?v=TdmpAy1hI8g talk by Tim May himself

Reply:
I have added both documentaries.

Plus this:

https://github.com/eth-protocol-fellows/protocol-studies/blob/a2fe6bd5b31754c6f2e7804db30a4930cc1776c4/docs/wiki/protocol/prehistory.md?plain=1#L47